src/shared/utils/encryption/decryptBASE64.ts

```diff
@@ -0,0 +1,3 @@
+export const decryptBASE64 = (value = '') => {
+  return window.atob(value);
```

Comment:
I don't think this will be working correctly since `window` object is not a part of React Native API
Please see https://stackoverflow.com/questions/42829838/react-native-atob-btoa-not-working-without-remote-js-debugging